content/next/index.md

```diff
@@ -96,7 +96,8 @@ The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL
 1. A scope MAY be provided after a type. A scope MUST consist of a noun describing a
   section of the codebase surrounded by parenthesis, e.g., `fix(parser):`
 1. A description MUST immediately follow the colon and space after the type/scope prefix.
-The description is a short summary of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
+The description is a short summary of the code changes, e.g., _fix: don't ignore consecutive spaces when parsing array._
+1. When the type field is a verb, the description MAY be an object or action the verb is applied to, only when the change is trivial or clear enough from the type, e.g., _revert: drop Node 6 from testing matrix._ Nontrivial changes, especially those of a  `fix` type, SHOULD describe the changeset itself rather than the undesirable behavior it removes.
```

Comment:
I think I'm more comfortable with an FAQ suggestion, that speaks to some of these recommendations. It seems like we could keep the formal spec text centered mainly on logic that could be adopted by a parser; whereas this recommendation is more a guideline for how one might go about writing commit messages ...

It almost makes me wonder if we'd do better to add a "related reading" section, rather than continuing to expand the FAQ.